open-bot.yml

```diff
@@ -289,7 +289,7 @@ rules:
       last_action_age: 26w # half a year
     actions:
       comment:
-        identifer: inactive-warning
+        identifier: inactive-warning
```

Comment:
Looks like these were probably a bug before, since it is spelled correctly upstream and other places in this file